### PR TITLE
FPMUL Verification

### PR DESCRIPTION
**Pull request by vincentchu12**

### CODE REVIEW

File: FPMUL/FPMUL_TB.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        # and send them to our 'model'
+        self.input_monitor = FPMUL_InputMonitor(dut, dut.Start, dut.Clk, callback=self.model)
+
+    def model(self, transaction):
+        if self.done:
+            A, B = transaction
+            product = A * B
+            self.expected_output.append((A, B, product))
+
+    def start(self):
```
Comment (vincentchu12) [inline FPMUL/FPMUL_TB.py]: Remove start and stop functions, not needed for internal state.

File: FPMUL/FPMUL_TB.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+class FPMUL_TB(object):
+    def __init__(self, dut):
+        """
+        
+        """
+
+        # Some internal state
+        self.dut = dut
+        self.stopped = False
```
Comment (vincentchu12) [inline FPMUL/FPMUL_TB.py]: Remove internal state

**Comment by ntwong0** [comment]
Wait I thought you were gonna focus on your midterm tomorrow

**Comment by vincentchu12** [comment]
Oh its not ready for merge, and I was procrastinating

**Comment by ntwong0** [comment]
I had trouble with the following error for a while, pinpointed it to FPMUL_OutputMonitor
     "Received a transaction but wasn't expecting anything"

The fix is to explicitly compare txn_valid against True, i.e. on line 20 of FPMUL_OutputMonitor.py:
`             if self.txn_valid is True:`

**Pull request merged**